meetings/create-long-term-room.py

```diff
@@ -2,11 +2,13 @@
 from os.path import join, dirname
 from dotenv import load_dotenv
 
-dotenv_path = join(dirname(__file__), "../.env")
+dotenv_path = join(dirname(__file__), '../.env')
 load_dotenv(dotenv_path)
 
-VONAGE_APPLICATION_ID = os.environ.get("VONAGE_APPLICATION_ID")
-VONAGE_APPLICATION_PRIVATE_KEY_PATH = os.environ.get("VONAGE_APPLICATION_PRIVATE_KEY_PATH")
+VONAGE_APPLICATION_ID = os.environ.get('VONAGE_APPLICATION_ID')
+VONAGE_APPLICATION_PRIVATE_KEY_PATH = os.environ.get('VONAGE_APPLICATION_PRIVATE_KEY_PATH')
+ROOM_DISPLAY_NAME = os.environ.get('ROOM_DISPLAY_NAME')
+EXPIRATION_DATE = '2023-01-30T00:47:04+0000'
```

Comment:
This expiration date is in the past. Can we make it dynamic, say 30 days from today?

Reply:
Again, I was following the examples in the other SDKs but I think this one is worth changing. I'll remove the environment variable and create something dynamic in the file itself.

Reply:
Have done this now

Reply:
can't we simply do the dynamic part in here so we can keep the variable?

Reply:
Not in python, maybe in bash? Though this isn't used anywhere else so I don't think it's a big deal to have this in the file

Reply:
I think having it in the `.env` file adds unnecessary complexity to that file, and showing python devs some arcane shell scripting isn't going to be as user-friendly as a clear python interpretation

Reply:
meant in this file, `create-long-term-room.py` and not in `.env`

Reply:
The variable isn't used for anything except for this one file, so I think it's fine to remove it. I can lowercase it in the file so it's clear that you don't need to set anything in the .env?

Reply:
Have made this change ^ now